### ISSUE TITLE
13 True Ways Classes

### ISSUE THREAD

**Issue by asacolips** [asacolips/toolkit13#2]
Add the 13 True Ways Classes to the system. This is waiting on an SRD version of these classes.

**Comment by asacolips** [comment]
SRD is getting closer to completion, see https://plus.google.com/+TimBaker/posts/MZbpBECE4xf

**Comment by Nine-Hands** [comment]
Seems like the SRD is still being worked on.  You have an alternate path to just use the OGL information directly from the source.  I did a quick read of the OGL declarations in 13 True Ways and it looks like all the class information/feats/abilities are open content and therefore available for use.  Obviously you must declare the use of that open content on the website but you should already be doing that when you quote from the SRD.

**Comment by asacolips** [comment]
That's true, and it may come to a point where that's what I have to do. I've been holding off on this because the fantastic writing and flavor text in 13th Age are really intertwined with the rules, and I don't want to misinterpret something as being OGL when it's actually considered flavor. The SRD makes it crystal clear what is and isn't OGL. On top of that, I haven't really had much time to devote to this in months, so any time that I do spend on it, I would want to put towards development/design where it's less ambiguous.

If you're interested in contributing to getting the powers/talents for the 13TW classes into the site, I would be more than happy to set up (or upgrade) your account to include permissions to make new powers/talents.

**Comment by sirglandon** [comment]
Does [this site](https://www.13thagesrd.com/) have the official SRD? What can I do to help incorporate the additional data?

**Comment by asacolips** [comment]
@sirglandon it does, and that's how I originally entered the content. I've since shifted focus to working on my [13th Age system for Foundry VTT](https://gitlab.com/asacolips-projects/foundry-mods/archmage), which has a larger portion of the content there (but not quite all of it). I'm planning to export the JSON data from that system into a format that I can use toolkit13.com in the future once it's more complete, but I could get you can account with create and edit own permissions for class content on toolkit13.com. We'll need to coordinate closely during that process so that I can review the content as it comes in to make sure the OGL is being adhered to for Pelgrane.

**Comment by sirglandon** [comment]
Perhaps you would rather I get things into JSON format. I don't do PHP. I'm more of a data guy. I know SQL very well, and JSON fairly well. I can also put stuff directly into toolkit13, or even onto a spreadsheet. Whatever works best for you, let me know. And your priorities.
